rosetta/retriever/retriever.go

```diff
@@ -324,6 +324,11 @@ func (r *Retriever) Transaction(block identifier.Block, id identifier.Transactio
 	}
 	for _, event := range events {
 
+		// ignore events belonging to other transactions
```

Comment:
This piece of code is left here, so that the retriever behaves as expected. When the other PR that fixes this same thing is merged, I could revert it, or just leave the tests failing in this PR until it's fixed. WDYT?